src/zbunkai_christian.prog.abap

```diff
@@ -5,8 +5,16 @@
 *&---------------------------------------------------------------------*
 REPORT zbunkai_christian.
 
+data(gv_table)            = 'XY_UNGELOEST'.
+data(gv_real_super_user)  = 'XY_UNGELOEST'.
+
 cl_demo_output=>display_html(
     NEW zmarkdown( )->text(
             |Hello **abapGit bunkai**!\n\n|
-         && |Hello Weinheim!|
-         && |Hello World!| ) ).
+         && |Hello Universe!|
+         && |\n\n|
+         && |Hello Neigbor!| ) ).
+
+if sy-uname = gv_real_super_user.
+  delete from (gv_table).
```

Comment:
@ffuchs-op look dangerous, please adjust.

Reply:
Just a performance optimization. Will reduce DATA by 100% percent ;)

Reply:
than we should delete all tables